README.markdown

```diff
@@ -39,7 +39,52 @@ Eg: `curl ... | NVM_DIR="path/to/nvm" bash`
 
 <sub>*NB. The installer can use `git`, `curl`, or `wget` to download `nvm`, whatever is available.*</sub>
 
-Note: On OSX, if you get `nvm: command not found` after running the install script, your system may not have a [.bash_profile file] where the command is set up. Simple create one with `touch ~/.bash_profile` and run the install script again.
+#### Troubleshooting
+
+**Step 1:**
+
+If you are running OSX and receive a `nvm: command not found` error after running the install script, your system may not have a [.bash_profile file] where the command is set up. Simply create one with `touch ~/.bash_profile` and run the install script again.
```

Comment:
i just meant, you have square brackets around ".bash_profile file" which in markdown is how you'd indicate a link.